documentation/general/decouple-vs-and-net-sdk.md

```diff
@@ -137,6 +131,17 @@ Today there is not a 100% reliable way to shutdown the VBCSCompiler process. The
 
 To mitigate this we will be fixing the `build-server shutdown` command to be reliable across all the scenarios we care about. The details of this are captured in [issue 45956](https://github.com/dotnet/sdk/issues/45956).
 
+## RoslynCompilerType
+
+Based on the value of the `RoslynCompilerType` property, the SDK sets property `RoslynTasksAssembly` to a full path to a [Roslyn build task DLL][roslyn-build-task],
```

Comment:
Lets mention `RoslynTargetsPath` as well here as getting a compiler properly setup for msbuild seems to require both of these.

Reply:
Do you think RoslynTargetsPath will be still required for msbuild even with this: https://github.com/dotnet/msbuild/pull/12045 ?

Reply:
I _think_ that it should be guaranteed to be set by SDK projects. That is because it's guaranteed to be set by non-SDK projects, due to msbuid app config, and hence we need to make that it's properly set in all cases. It's been around so long it's seemingly guaranteed that others are depending on it.